karma.config.cjs

```diff
@@ -0,0 +1,53 @@
+const webpackConfig = require('./webpack.config.js')
+
+module.exports = function (config) {
+  config.set({
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+    frameworks: ['webpack', 'jasmine'],
+    // list of files / patterns to load in the browser
+    files: [
+      'karma.setup.cjs',
+      '__tests__/integration/*.test.ts',
+      '__tests__/integration/browser/*.test.ts',
+      '__tests__/utils/*.ts',
+      '__tests__/unit/*.test.ts',
+      '__tests__/unit/browser/*.test.ts',
+    ],
+    exclude: [],
+    webpack: webpackConfig,
+    preprocessors: {
+      'packages/client-common/**/*.ts': ['webpack', 'sourcemap'],
+      'packages/client-browser/**/*.ts': ['webpack', 'sourcemap'],
+      '__tests__/unit/*.test.ts': ['webpack', 'sourcemap'],
+      '__tests__/unit/browser/*.test.ts': ['webpack', 'sourcemap'],
+      '__tests__/integration/*.ts': ['webpack', 'sourcemap'],
+      '__tests__/integration/browser/*.ts': ['webpack', 'sourcemap'],
+      '__tests__/utils/*.ts': ['webpack', 'sourcemap'],
+    },
+    reporters: ['progress'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: false,
+    // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
+    browsers: ['Chrome_without_security'],
```

Comment:
Just curious: are we going to test in Firefox in the future?

Reply:
We should, I think. There are differences in streaming support. Once this one is close to the release, let's create the issues for leftovers.

Reply:
A follow-up: https://github.com/ClickHouse/clickhouse-js/issues/179